packages/firebase_firestore/lib/src/document_snapshot.dart

```diff
@@ -12,8 +12,9 @@ part of firebase_firestore;
 class DocumentSnapshot {
   /// Contains all the data of this snapshot
   final Map<String, dynamic> data;
+  final String path;
```

Comment:
This field should be documented.

The Java API has a `getId` for the ID of the document. Should we include that too?

[nit] As `data` logically depends on `path`, I think we should swap the order of `data` and `path`, both here and in the constructor.